ebos/eclwellmanager.hh

```diff
@@ -567,6 +568,10 @@ public:
         return wellDat;
     }
 
+    Opm::data::Group groupData(const int reportStepIdx, Opm::Schedule& sched) const {
```

Comment:
The parameters here are unused.  I suggest writing the declaration as
```C++
Opm::data::Group groupData(const int /* reportStepIdx */, const Opm::Schedule& /* sched */) const {
```
In other words, without naming the parameters.  Doing so will avoid warnings about unused parameters.

Reply:
Thank you for the comment, I agree and this change has now been done and pr has been updated.

Reply:
> Thank you for the comment, I agree and this change has now been done and pr has been updated.

Thank you very much.  Did you want to address my other comment too, or do you consider this PR done now?